rng/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/local/repos/GH/sknorr/dietrich/rng/conversion.rnc" type="application/relax-ng-compact-syntax"?>
+<config version="1.0">
+ <property key="STLYEROOT" value="/usr/share/xml/docbook/stylesheet/suse2013"/>
```

Comment:
using a style more like `<styleroot>/usr/...</styleroot>` for all the properties would make this easier to validate imo.

Reply:
Ok, integrated.